### PR TITLE
Hi

### PR DESCRIPTION
**Pull request by chylaudes**

### CODE REVIEW

**Comment by tgaff** [comment]
@chylaudes ?? :rabbit:

**Comment by chylaudes** [comment]
:cow:

**Comment by Borgaard** [comment]
:octopus:

**Comment by Borgaard** [comment]
Do not merge this PR pls. I'm... testing... something...

File: ChristineLau-chylaudes/hw_test/test.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def hello
```
Comment (amblount) [inline ChristineLau-chylaudes/hw_test/test.rb]: Can you see this?

Comment (amblount) [inline ChristineLau-chylaudes/hw_test/test.rb]: No error message

Comment (Borgaard) [inline ChristineLau-chylaudes/hw_test/test.rb]: NICE! Thank you, Alivia! :octocat:

**Comment by Borgaard** [comment]
Tested! Success cactus :cactus:

File: ChristineLau-chylaudes/hw_test/test.rb (hunk 1 above)
Comment (rayliaw) [inline ChristineLau-chylaudes/hw_test/test.rb]: :cake:

Comment (tgaff) [inline ChristineLau-chylaudes/hw_test/test.rb]: :+1:

Comment (chinhle94115) [inline ChristineLau-chylaudes/hw_test/test.rb]: the test works.

**Pull request merged**